codegen/templates/templates.go

```diff
@@ -493,6 +492,7 @@ var commonInitialisms = map[string]bool{
 	"XMPP":  true,
 	"XSRF":  true,
 	"XSS":   true,
+	"ZIP":   true,
```

Comment:
I worry that `ZIP` might fall into the category of words that are _not_ [highly unlikely to be initialisms](https://github.com/99designs/gqlgen/pull/1877/files#diff-54e0b04129f7e9f7a0d75ae045a49f4c09d5cba9d09fb01fc499719216bb5b86R447). It may also refer to the term in compression and textiles contexts.